ziti/src/main/kotlin/org/openziti/impl/ChannelImpl.kt

```diff
@@ -94,7 +94,7 @@ internal class ChannelImpl(val addr: String, val id: Identity, val apiSession: (
         return async { chState.filter { it is Channel.State.Connected }.first() }
     }
 
-    @ExperimentalCoroutinesApi
+//    @ExperimentalCoroutinesApi
```

Comment:
verifying that this is intentionally commented out. if so -- should it just be removed?